app/api/models/errors/ErrorWrapper.scala

```diff
@@ -26,7 +26,7 @@ case class ErrorWrapper(correlationId: String, error: MtdError, errors: Option[S
 
   private def allErrors: Seq[MtdError] = errors match {
     case Some(seq) => seq
-    case None      => Seq(error)
+    case None      => List(error)
```

Comment:
What's the motivation for making this a list? Using [Vectors](https://stackoverflow.com/questions/6928327/when-should-i-choose-vector-in-scala) instead might improve performance.

Reply:
As a team, we have decided to use List instead of Seq when providing a Seq/List type as a parameter within a function/method or using it within one. Explanation given by @SamuelJacobsHMRC  : "When instantiating Seqs/List, always use List. A Seq as a parameter type will accept vectors or lists Seqs are good in function definitions for keeping things generally and polymorphic. However, when we instantiate a Seq in Scala (i.e. val x = Seq(1,2,3)) it actually defaults to a List (as Seq is an abstract type that cant actually be instantiated) so it is more accurate to write a val is a List (i.e. val y = List(1,2,3)) than a Seq"

Reply:
That explains why List > Seq, but I don't think it addresses whether List > Vector. 

Vectors use less memory than Lists, so in cases where we don't care too much about what's in the middle of the list it seems like vectors may be preferable. 

It's fine as it is, I might ask about it in retro 👍

Reply:
Yep, to clarify we haven't decided to use List, it's more "When instantiating a type, specify the concrete class." If Vector's more suitable, then definitely use Vector!